lib/matplotlib/tests/test_usetex.py

```diff
@@ -14,7 +15,7 @@ def usetex():
 
 @image_comparison(baseline_images=['test_usetex'],
                   extensions=['pdf', 'png'],
-                  tol=0.3)
+                  tol={'aarch64': 2.868}.get(platform.machine(), 0.03))
```

Comment:
Previous was 0.3, not 0.03. Not sure this was intentional.

Reply:
Oops, it's not, sorry.  I'll fix it up right away.